locales/en/translation.json

```diff
@@ -60,7 +61,10 @@
         "css": "CSS",
         "javascript": "JS"
       },
-      "output": "Output"
+      "output": "Output",
+      "instructions": {
+        "cancel": "Cancel"
```

Comment:
Not quite sure why this string is separate from "`add-or-edit-instructions"`.
Couldn't we do...
```
"instructions": {
    "add": "Add Instructions",
    "edit": "Edit Instructions",
    "cancel": "Cancel"
}
```

Reply:
Purposely omitted `"Save"` because you haven't accomplished that yet, but same deal as with the other strings.

Reply:
I separated them because `add-or-edit-instructions` (soon to be split into two separate strings) is in the **`top-bar`**, while `cancel` and `save` are in the `Instructions` **component**. But I don't feel strongly about that separation, so I'm happy to group them all together if you think that would be better.